main.go

```diff
@@ -51,6 +51,7 @@ const (
 	autoInstrumentationDotNetImageRepository = "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-dotnet"
 	dcgmExporterImageRepository              = "nvcr.io/nvidia/k8s/dcgm-exporter"
 	neuronMonitorImageRepository             = "public.ecr.aws/neuron"
+	targetAllocatorImageRepository           = "ghcr.io/open-telemetry/opentelemetry-operator/target-allocator"
```

Comment:
Relying on upstream image until https://github.com/aws/amazon-cloudwatch-agent-operator/pull/214 is released as an image.